docs/planning/backend/app/api/fragrances.py

```
@@ -0,0 +1,210 @@
+
+    # Build response
+    response = FragranceResponse.model_validate(fragrance)
+    response.top_notes = [NoteResponse.model_validate(n) for n in notes_by_position['top']]
+    response.heart_notes = [NoteResponse.model_validate(n) for n in notes_by_position['heart']]
+    response.base_notes = [NoteResponse.model_validate(n) for n in notes_by_position['base']]
+
+    return response
+
+
+@router.post("/", response_model=FragranceResponse, status_code=201)
+def create_fragrance(data: FragranceCreate, db: Session = Depends(get_db)):
```

Ruff (0.14.10)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)